source/get-started.txt

```diff
@@ -46,8 +46,16 @@ customize. The following template apps are available:
 - An Event-driven :doc:`Database Trigger </triggers/database-triggers>` 
   template that updates a view in a separate collection.
 
-Checkout the template apps or visit the Tutorials section for a walkthrough
-of a template app.
+To learn more:
+
+- Watch the `Learning Byte on Template Apps 
+  <https://learn.mongodb.com/courses/using-template-apps-in-the-atlas-ui>`__.
```

Comment:
this takes you to the main landing page for the LB (the overview?), but the other links land you on the first video. Do we want to be consistent with where we're linking in the LB?